test/NVQPP/mapping_test-2.cpp

```diff
@@ -6,8 +6,8 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// RUN: nvq++ -v %s -o %t --target oqc --emulate && CUDAQ_DUMP_JIT_IR=1 %t |& FileCheck %s
-// RUN: nvq++ -std=c++20 --enable-mlir %s -o %t
+// RUN: nvq++ %cpp_std -v %s -o %t --target oqc --emulate && CUDAQ_DUMP_JIT_IR=1 %t |& FileCheck %s
+// RUN: nvq++ -std=c++17 --enable-mlir %s -o %t
```

Comment:
I am assuming the second line here was intended to be c++17 rather than fixing it at c++20.

Reply:
I believe that came from @schweitzpgi as part of #973. I don't know for sure, but I suspect he threw one `c++20` in there just to make sure that it worked when specifying the newer verison (and not just the older version).

Reply:
Let me keep that then, provided C++20 is available.